public/js/reducers/sources.js

```diff
@@ -13,15 +13,13 @@ import type { Record } from "../utils/makeRecord";
 export type SourcesState = {
   sources: I.Map<string, any>,
   selectedSource: ?any,
-  sourcesText: I.Map<string, any>,
   tabs: I.List<any>,
   sourceMaps: I.Map<string, any>,
 };
 
 const State = makeRecord(({
   sources: I.Map(),
   selectedSource: undefined,
-  sourcesText: I.Map(),
```

Comment:
`sourcesText` was moved to a field in `source`. This simplified some of the pretty print explorations and afterwards seemed like a cleaner abstraction.